plugins/importexport/doaj/filter/DOAJJsonFilter.inc.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+/**
+ * @file plugins/importexport/doaj/filter/DOAJJsonFilter.inc.php
+ *
+ * Copyright (c) 2014-2016 Simon Fraser University Library
+ * Copyright (c) 2000-2016 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class DOAJJsonFilter
+ * @ingroup plugins_importexport_doaj
+ *
+ * @brief Class that converts an Article to a DOAJ JSON string.
+ */
+
+import('lib.pkp.plugins.importexport.native.filter.NativeImportExportFilter');
+
+
+class DOAJJsonFilter extends NativeImportExportFilter {
+	/**
+	 * Constructor
+	 * @param $filterGroup FilterGroup
+	 */
+	function __construct($filterGroup) {
+		$this->setDisplayName('DOAJ JSON export');
```

Comment:
For some reason all Filters have hard coded names that are then saved in the DB. It seems to me that is OK so i.e. I am not sure if it makes sense to translate them i.e. to have them in the locale files? Else, maybe we should not store them in the DB? Where else is the name used?